Rest & API: Remove web.py; Fix #4044

bziemons opened the pull request:
Remove all endpoints and related unused code.
Switch all configuration to use flask and create stub main.py entrypoints to use.

rcarpa reviewed (commented):
Looks good, but I don't know all this code very well.

bziemons committed:
Rest & API: Remove web.py; Fix #4044

Remove all endpoints and related unused code.
Switch all configuration to use flask and create stub main.py
entrypoints to use.

bari12 commented:
LGTM, @tbeerman please also have a look on this.

bziemons commented:
WebUI seems broken. Investigating

Update: it was "just" the dev container where the WebUI is broken. Pushing a fix within this PR here.
